packages/core/src/styles/britecharts.scss

```diff
@@ -1,24 +1,24 @@
 // Base Styles
-@import 'base/_typography';
+@import "base/_typography";
 
 // Helpers
-@import 'helpers/_variables';
-@import 'helpers/loadingStates';
+@import "helpers/_variables";
+@import "helpers/loadingStates";
```

Comment:
why changing to double quotes?

Reply:
Not sure why (I think a bug related to the three stylelint recommendation sets we were loading) I had to change slightly the stylelint settings. This was part of it. 
I really don´t care as long as they get fixed automatically!